examples/3D_pipe_flow/case.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+import math
```

Comment:
are you sure this case is working as well as it was before you introduced the BC changes? can you run the same case with the master branch and compare the result? you did have to regenerate the goldenfiles it looks like...

Reply:
I can approximate these cases on the master branch. Without the new inflow BCs, the comparison won't be exactly the same. I think the reason I had to regenerate the golden files was that the old implementation didn't take into account the effect assigning momentum had on the energy equation. I'll try running this with a characteristic inflow and a small non-zero velocity region near the inlet and see how it compares.